### PR TITLE
grammar fix for untrusted source message

### PR DESCRIPTION
**Pull request by Laevos**
Should be "likely forwarded" instead of "like forwarded"

### CODE REVIEW

File: resources/qml/EncryptionIndicator.qml [hunk 1]
```diff
@@ -41,7 +41,7 @@ Image {
         case Crypto.TOFU:
             return qsTr("Encrypted by an unverified device, but you have trusted that user so far.");
         case Crypto.MessageUnverified:
-            return qsTr("Key is from an untrusted source like forwarded from another user or the online key backup. For this reason we can't verify who sent the message.");
```
Comment (deepbluev7) [inline resources/qml/EncryptionIndicator.qml:44]: So, my english sucks, but my assumption was that wording it like this means basically something like "Key is from an untrusted source. Examples of that could be, that the key was forwarded by another user or that the key is downloaded from the online key backup". I think changing this to "likely" actually loses that meaning somewhat? @LorenDB or @redsky17 , could you check what makes more sense?

**Comment by redsky17** [comment]
@deepbluev7 I agree with you.  I think “likely” does lose some meaning.  That being said, I do think the wording of the original is a little ambiguous and can be read the other way.  I feel like “such as” might be a decent replace for “like” in the original

**Comment by deepbluev7** [comment]
Sorry, but I don't know what that command means.

Thanks for asking 🥰

**Comment by deepbluev7** [comment]
Sorry, but I don't know what that command means.

Thanks for asking 🥰

File: resources/qml/EncryptionIndicator.qml (hunk 1 above)
Comment (LorenDB) [reply resources/qml/EncryptionIndicator.qml:44]: I agree with what @redsky17 said. I'd personally go with something like "Key is from an untrusted source, possibly forwarded from another user or the online key backup. For this reason we can't verify who sent the message.".

**Review by deepbluev7 — APPROVED** [review]
Thank you!

**Pull request merged**